requirements.txt

```diff
@@ -1 +1,5 @@
 flask
+pandas
+math
+numpy
+pathlib
```

Comment:
Since pathlib is part of the standard library, it does not need to be a dependency